hedyorg/hedy
# 🪲 Add songs and hotel adventures to level 13 #6008

Fixes #6008

**How to test**
Open level 13 and check there are tabs for the `Sing a song` and `Hotel` adventures. These should appear in the old UI and in the new one (tryit/13). Also, in the new UI, we should be able to navigate to these adventures from the sidepanel on the left.
